### PR TITLE
Adds documentation for #p

### PR DESCRIPTION
**Pull request by lukeraymonddowning**

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]

File: documentation.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
+    - [Custom Helpers](/docs/helpers)
+    - [IDE Plugins](/docs/ide-plugins)
+    - [Exceptions & Errors](/docs/exceptions-and-errors)
+    - [Groups Of Tests](/docs/groups)
+    - [Skipping Tests](/docs/skipping-tests)
+    - [Datasets](/docs/datasets)
+    - [Coverage](/docs/coverage)
+    - [CLI Options](/docs/cli-options)
+- ## Plugins 
+    - [Faker](/docs/plugins/faker)
```
Comment (AlexMartinFR) [inline documentation.md:18]: It's a nice touch to reorder the plugin in alphabetical order!
However, the internal linking will be broken unless every concerned doc page is edited to reflect the new order.

Comment (lukeraymonddowning) [reply documentation.md]: Reverted the menu 👍

Comment (lukeraymonddowning) [reply documentation.md]: We can update that in a separate PR

File: plugins/parallel.md [hunk 2]
```diff
@@ -18,6 +19,16 @@ you can drastically reduce the time it takes to run large test suites.
 
 **Source code**: [github.com/pestphp/pest-plugin-parallel](https://github.com/pestphp/pest-plugin-parallel)
 
+<a name="requirements"></a>
+## Requirements
```
Comment (nunomaduro) [inline plugins/parallel.md:23]: I don't get this section. IMO this requirements are ensured by composer itself, via the conflict or requirements section.

Regarding Laravel, maybe throw an exception on the plugin itself if the version is not ^v8.55.0?

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: plugins/parallel.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- [Overview](#overview)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Laravel](#laravel)
+
+<a name="overview"></a>
+## Overview
+
+Pest is the first testing framework to ship with first-party support for parallel testing. Using this feature,
```
Comment (nunomaduro) [inline plugins/parallel.md:17]: @lukeraymonddowning Maybe we can rewrite this differently - something like:

```
By default, Pest execute your tests sequentially within a single process. However, you may greatly reduce the amount of time it takes to run your tests by using the Parallel plugin that allows you run tests simultaneously across multiple processes. 
```

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: plugins/parallel.md (hunk 2 above)
Comment (lukeraymonddowning) [reply plugins/parallel.md]: Removed 👍

File: plugins/parallel.md (hunk 3 above)
Comment (lukeraymonddowning) [reply plugins/parallel.md]: Updated

**Pull request merged**